types/wlr_data_device.c

```diff
@@ -359,7 +340,8 @@ static void data_device_set_selection(struct wl_client *client,
 		wl_resource_get_user_data(dd_resource);
 
 	// TODO: store serial and check against incoming serial here
```

Comment:
I think we decided that serial validation is not necessary here so we can remove these todos.